src/Internal/Requirements/GoogleProductFeedValidator.php

```diff
@@ -0,0 +1,100 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\Internal\Requirements;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\Exception\ExtensionRequirementException;
+use Automattic\WooCommerce\GoogleListingsAndAds\MerchantCenter\MerchantStatuses;
+use Automattic\WooCommerce\GoogleListingsAndAds\Options\TransientsInterface;
+use Automattic\WooCommerce\GoogleListingsAndAds\PluginHelper;
+use DateTime;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class GoogleProductFeedValidator
+ *
+ * @since x.x.x
+ *
+ * @package AutomatticWooCommerceGoogleListingsAndAdsInternalRequirements
+ */
+class GoogleProductFeedValidator extends RequirementValidator {
+
+	use PluginHelper;
+
+	/**
+	 * Validate all requirements for the plugin to function properly.
+	 *
+	 * @return bool
+	 */
+	public function validate(): bool {
+		try {
+			$this->validate_google_product_feed_inactive();
+		} catch ( ExtensionRequirementException $e ) {
+
+			add_filter(
+				'woocommerce_gla_account_issues',
+				function( $issues, $current_time ) {
+					return $this->add_conflict_issue( $issues, $current_time );
+				},
+				10,
+				2
+			);
+
+			add_filter(
```

Comment:
Is this really a filter or an action? If it would be a filter we need to remember to return a value from the function.

Reply:
Good catch, [definitely an action](https://developer.wordpress.org/reference/hooks/deactivated_plugin/) d953023720e1c7481095801c02aebebf15ba5e23